x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/rules_table/rules_table_auto_refresh.cy.ts

```diff
@@ -30,10 +30,9 @@ import { createRule } from '../../../../tasks/api_calls/rules';
 import { cleanKibana } from '../../../../tasks/common';
 import { getNewRule } from '../../../../objects/rule';
 
-const DEFAULT_RULE_REFRESH_INTERVAL_VALUE = 60000;
```

Comment:
Let's keep the previous naming. In this context, `DEFAULT_RULE_REFRESH_INTERVAL_VALUE` provides more clarity than `ONE_MINUTE_IN_MS`